Add spluk_cloud-platform, splunk_universal-forwarder and anaconda files (#104)
odidev/ecosystem-dashboard-for-arm: content/commercial_packages/anaconda.md

---
name: Anaconda
vendor: Anaconda
category: Data Science & ML

> description: Anaconda software helps you create an environment for many different versions of Python and package versions. Anaconda is also used to install, remove, and upgrade packages in your project environments.

Author: ```suggestion
description: Anaconda software helps creating an environment for many different versions of Python and package versions. Anaconda is also used to install, remove, and upgrade packages in the project environments.
```
